### PR TITLE
Adds more sanity checks to Fusion, and a temperature check to react.

### LINKED ISSUE #13971: atmos is incredably broken
You  use the analyzer on  the canister [Plasma].
Results  of analysis of  the canister [Plasma].
Pressure: 34.1 kPa
Nitrogen: -315 %
Oxygen: 106 %
CO2: 110 %
Plasma: 197 %
Temperature: -240 °C
this is after canister ignition of burnmix. 66/33 plasma/ox
imedeatly befor i heated it to this tempature it was -1400000 k and psi

### PR DESCRIPTION
**Pull request by as334**
I think at this point we have sanity checks on everything we possibly can, so we should be good.

Hopefully fixes #13971

### CODE REVIEW

**Comment by as334** [comment]
#MostBrokenFeature2015

File: code/datums/gas_mixture.dm [hunk 1]
```diff
@@ -104,7 +104,8 @@ What are the archived variables for?
 
 /datum/gas_mixture/proc/react(atom/dump_location)
 	var/reacting = 0 //set to 1 if a notable reaction occured (used by pipe_network)
-
+	if(temperature < TCMB)
+		temperature = TCMB
```
Comment (duncathan) [inline code/datums/gas_mixture.dm]: Temperature = max(temperature, TCMB)

Comment (duncathan) [inline code/datums/gas_mixture.dm]: Actually scratch that, I'd rather not have another proc call in here

Comment (as334) [inline code/datums/gas_mixture.dm]: that was my thoughts yeah

**Comment by duncathan** [comment]
Alright he says it compiles and this is high priority

**Comment by Chocobro** [comment]
those label colours are aesthetic
:+1:

**Pull request merged**